src/app/loadout-drawer/loadout-utils.ts

```diff
@@ -786,11 +786,11 @@ function isFashionPlug(defs: D2ManifestDefinitions, modHash: number) {
 export function getModsFromLoadout(
   defs: D2ManifestDefinitions | undefined,
   loadout: Loadout,
-  unlockedPlugs: Set<number>,
+  unlockedPlugs?: Set<number>,
```

Comment:
```suggestion
  unlockedPlugs = new Set<number>(),
```

Kinda makes it clearer?